notebooks/211-speech-to-text/211-speech-to-text.ipynb

```diff
@@ -0,0 +1,559 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_folder = \"model\"\n",
+    "download_folder = \"output\"\n",
+    "data_folder = \"data\"\n",
+    "\n",
+    "precision = \"FP16\"\n",
+    "model_name = \"quartznet-15x5-en\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download and Convert Public Model\n",
+    "\n",
+    "We use `omz_downloader` and `omz_converter`, which are command-line tools from the `openvino-dev` package. \n",
+    "\n",
+    "If it is your first run, models will download and convert here. It might take up to ten minutes. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d946d37",
+   "metadata": {},
+   "source": [
+    "### Download Model\n",
+    "\n",
+    "`omz_downloader` automatically creates a directory structure and downloads the selected model. This step is skipped if the model is already downloaded. The selected model comes from the public directory, which means it must be converted into Intermediate Representation (IR)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check if model is already downloaded in download directory\n",
+    "if not path.isdir(f'{download_folder}/public/{model_name}'):\n",
+    "    download_command = f\"omz_downloader --name {model_name} --output_dir {download_folder} --precision {precision}\"\n",
+    "    ! $download_command\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8c0a5cdb",
+   "metadata": {},
+   "source": [
+    "### Convert Model\n",
+    "\n",
+    "`omz_converter` is needed to convert pre-trainded `PyTorch` model to ONNX model format, which is further converted to OpenVINO IR format. Both stages of conversion are handeled by calling `omz_converter`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8cfaf7ee",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check if model is already converted in model directory\n",
+    "\n",
+    "if not path.isdir(f'{model_folder}/public/{model_name}/{precision}'):\n",
+    "    convert_command = f\"omz_converter --name {model_name} --precisions {precision} --download_dir {download_folder} --output_dir {model_folder}\"\n",
+    "    ! $convert_command"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Audio Processing\n",
+    "\n",
+    "Now, when model is loaded, you need to load audio file. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c4ad393",
+   "metadata": {},
+   "source": [
+    "### Defining constants\n",
+    "\n",
+    "First step will be locating audio file and defining alphabet used by model. In this case you will use latin alphabet begining with space symbol and ending with blank symbol, in our case it will be `~`, but that could be any other char."
```

Comment:
```suggestion
    "First step is locating the audio file and defining alphabet used by the model. In this case, you will use latin alphabet beginning with a space symbol and ending with a blank symbol, in our case it will be `~`, but that could be any other char."
```